docs/user/azure/dnszone.md

```diff
@@ -0,0 +1,31 @@
+# Public DNS Zone
+
+Azure's DNS Zone service is used by the OpenShift installer to configure cluster DNS resolution and provide name lookup
+for the cluster to the outside world. To use OpenShift, you must have created a public DNS zone in Azure in
+the same subscription as your OpenShift cluster. You must also ensure the zone is "authoritative" for the domain. There are
+two ways to do this outlined below: root domain and subdomain. A root domain is `openshiftcorp.com`. A subdomain is of
+the form `clusters.openshiftcorp.com`.
```

Comment:
I prefer `example.com` based names, e.g. [here][1] (discussed in the bfc40f87fbdbc commit message).

[1]: https://github.com/openshift/installer/pull/1393/files#diff-ed7e942cdd77790bfef1b7ae305fd119L15

Reply:
https://github.com/openshift/installer/compare/fc6532c8637a2c37f4c0a028da1a5a430ac19302...3bb0cfefaf513837f94ff94c1bc59e047421323b